### ISSUE TITLE
ApiException without details

### ISSUE THREAD

**Issue by tmirkes** [cezarykluczynski/stapi#45]
When attempting to retrieve an EpisodeFullResponse using a String UID value acquired via an EpisodeBaseResponse, I receive an ApiException, but I have no idea why.

I iterate over the EpisodeBase objects returned from the search, and when one matches my desired condition, I assign its UID to a String variable.  (I've confirmed that the value is being properly populated to the variable, and that it is correctly typed to String.)  Then that String variable is used as the parameter on the get() method of an Episode object.

I have been unable to get anything but an ApiException from this step of my process.  It doesn't matter if I follow the example in the Java Client documentation (using hardcoded UID text) or supply a String variable containing the confirmed UID value; the result is the same.

Is there a way to get more detailed error messages from the ApiException to help with debugging?

(My apologies if I'm missing something obvious; I'm new to working with APIs, and there doesn't seem to be much documentation or tutorial material out there for this API.  Please let me know if I'm asking in the wrong place or to the wrong people.)

**Comment by cezarykluczynski** [comment]
You're quite correct in your investigation, and the steps you described should give the results you expect.

I identified this problem two weeks ago, and a fix is already commited to the repository, along a new layer of performance tests that make sure everything is loading correctly, and a new caching strategy that would make problematic entities like episodes load a lot faster. New release of stapi.co is brewing, and will happen this week, probably even tomorrow.

As for the more detailed description of ApiException, however, this has more to do with how code is generated by Swagger codegen, and there is no easy way of fixing this that I know of. But I'll make a note to look into it at some point.

**Comment by cezarykluczynski** [comment]
@tmirkes New version was deployed and there shouldn't be a problem with loading full episodes anymore.

Please note that data was freshly loaded, so as of March there are now 855 episodes, not 852. :wink:

**Comment by tmirkes** [comment]
@cezarykluczynski Thank you so much!  I've just tested my code, and things look to be running smoothly (after updating my tests to check for the appropriate number of episodes, of course).
